broker-node/app/Clients/BrokerNode.php

```diff
@@ -227,21 +221,18 @@ private static function sendToHookNode(&$chunks, $request)
 
         NodeMessenger::sendMessageToNodesAndContinue($tx, $hookNodes);
 
-        //record event
-        Segment::track([
-            "event" => "chunk_sent_to_hook",
-            "properties" => [
-                "broker_url" => $_SERVER['REMOTE_ADDR'],
-                "hooknode_url" => $hookNodeUrl,
-            ]
-        ]);
-
-        // DEPRECATED. This will be replaced with segment.io
-        self::initEventRecord();
-        self::$ChunkEventsRecord->addChunkEvent("chunk_sent_to_hook", $hookNodeUrl, "todo", "todo");
         HookNode::incrementChunksProcessed($hookNodeUrl, count($chunks));
 
         array_walk($chunks, function ($chunk) use ($hookNodeUrl, $request) {
+            //record event
+            Segment::track([
```

Comment:
Since we send this as an array, should we log just 1 event with an array of `chunk_idxs`? It would more accurately reflect what we send to the hooknode.